mono/utils/mono-dl-wasm.c

```diff
@@ -45,7 +46,7 @@ mono_dl_get_system_dir (void)
 void*
 mono_dl_lookup_symbol (MonoDl *module, const char *name)
 {
-	return NULL;
+	return dlsym(module->handle, name);
 }
```

Comment:
These should use mono calling conventions (space after ( ).

Reply:
I'll try to add the .editorconfig file so this does not happen anymore... you've got to be tired of those 
 :)